department-of-veterans-affairs/caseflow
# Fix pagination appeal type sort order

Resolves bug where paginated queues did not bubble aod cases to the top

### Description
Fixed task sorting on the back end to always sort aod to the top when sorting by appeal type.

### Acceptance Criteria
- [ ] When pagination is enabled, the default sort order on page load bubbles AOD cases to the top

### Testing Plan
1. Ensure user queue pagination is enabled
    ```ruby
    FeatureToggle.enable!(:user_queue_pagination, users: [User.find_by(css_id: "CSS_ID1").css_id])
    ```
1. On master, log in as CSS_ID1 and go to their queue.
1. Notice the AOD cases are not listed first
1. pull this branch and reload
1. Notice AOD cases listed first

### User Facing Changes

 BEFORE|AFTER
 ---|---
![Screen Shot 2020-05-28 at 11 54 06 AM](https://user-images.githubusercontent.com/45575454/83166517-0c221c00-a0dd-11ea-99df-7a4946a906a9.png)|![Screen Shot 2020-05-28 at 12 00 23 PM](https://user-images.githubusercontent.com/45575454/83166526-0debdf80-a0dd-11ea-95d3-3df7ebe05f60.png)
